src/components/Navbar.tsx

```diff
@@ -1,4 +1,9 @@
+import { NavLink } from 'react-router-dom';
+
 export const Navbar = () => {
+  const isActiveLink = ({ isActive }: { isActive: boolean }) =>
+    `navbar-item ${isActive ? 'has-background-grey-lighter' : ''}`;
```

Comment:
- To set classes conditionally use [classnames](https://www.npmjs.com/package/classnames) or [clsx](https://www.npmjs.com/package/clsx) library
- Name doesn't describe what the function actually does